scipy/_build_utils/npy_1_18_compat/numpy/random/bitgen.h

```diff
@@ -0,0 +1,22 @@
+/* Vendor numpy/random/bitgen.h for numpy<=1.17 */
```

Comment:
Should this be <= 1.17?

Reply:
It should be for any environment with numpy < 1.18, so <= 1.17, yes.  This is in addition to random.pxd, I couldn't find an easy way to get pxds included from this directory.